splunk_handler/__init__.py

```diff
@@ -33,11 +33,14 @@ def __init__(self, host, port, username, password, index,
 
         # prevent infinite recursion by silencing requests and urllib3 loggers
         logging.getLogger('requests').propagate = False
-        logging.getLogger('urllib3').propagate = False
 
         # and do the same for ourselves
         logging.getLogger(__name__).propagate = False
 
+        # disable all warnings from urllib3 package
+        if not self.verify:
+            requests.packages.urllib3.disable_warnings()
```

Comment:
As far as I can tell, the default warning type disabled in disable_warnings() is HTTPWarning in this file: https://github.com/kennethreitz/requests/blob/master/requests/packages/urllib3/exceptions.py which seems to be the base warning class for all urllib3 warning objects.